lib/src/test/java/uk/gov/di/ipv/core/library/service/ConfigurationServiceTest.java

```diff
@@ -79,16 +84,58 @@ void shouldGetCredentialIssuerFromParameterStore() {
 
         Map<String, String> credentialIssuerParameters =
                 Map.of("tokenUrl", TEST_TOKEN_URL, "credentialUrl", TEST_CREDENTIAL_URL);
-        when(ssmProvider.getMultiple("/dev/IPV/Core/CredentialIssuers/PassportCRI"))
+        when(ssmProvider.getMultiple("/dev/ipv/core/credentialIssuers/passportCri"))
                 .thenReturn(credentialIssuerParameters);
 
-        CredentialIssuerConfig result = configurationService.getCredentialIssuer("PassportCRI");
+        CredentialIssuerConfig result = configurationService.getCredentialIssuer("passportCri");
 
         CredentialIssuerConfig expected =
                 new CredentialIssuerConfig(
-                        "PassportCRI", URI.create(TEST_TOKEN_URL), URI.create(TEST_CREDENTIAL_URL));
+                        "passportCri",
+                        "",
+                        URI.create(TEST_TOKEN_URL),
+                        URI.create(TEST_CREDENTIAL_URL),
+                        URI.create(TEST_CREDENTIAL_URL));
 
         assertEquals(expected.getTokenUrl(), result.getTokenUrl());
         assertEquals(expected.getCredentialUrl(), result.getCredentialUrl());
     }
+
+    @Test
+    void shouldGetAllCredentialIssuersFromParameterStore() {
+
+        environmentVariables.set("ENVIRONMENT", "dev");
+        HashMap<String, String> response = new HashMap<>();
+        response.put("passportCri/tokenUrl", "passportTokenUrl");
+        response.put("passportCri/authorizeUrl", "passportAuthUrl");
+        response.put("passportCri/id", "passportAuthid");
+        response.put("passportCri/name", "passportIssuer");
+        response.put("stubCri/tokenUrl", "stubTokenUrl");
+        response.put("stubCri/authorizeUrl", "stubAuthUrl");
+        response.put("stubCri/id", "stubAuthId");
+        response.put("stubCri/name", "stubIssuer");
+
+        when(ssmProvider.recursive()).thenReturn(ssmProvider2);
+        when(ssmProvider2.getMultiple("/dev/ipv/core/credentialIssuers")).thenReturn(response);
+        List<CredentialIssuerConfig> result = configurationService.getCredentialIssuers();
+
+        assertEquals(2, result.size());
+    }
+
+    @Test
+    void shouldGetAllCredentialIssuersFromParameterStoreNewAndIgnoreInexistingFields() {
+
+        environmentVariables.set("ENVIRONMENT", "dev");
+        HashMap<String, String> response = new HashMap<>();
+        response.put("passportCri/tokenUrl", "passportTokenUrl");
+        response.put("stubCri/tokenUrl", "stubTokenUrl");
+        // This will be ignored - not in pojo
+        response.put("stubCri/ipclientid", "stubIpClient");
+
+        when(ssmProvider.recursive()).thenReturn(ssmProvider2);
+        when(ssmProvider2.getMultiple("/dev/ipv/core/credentialIssuers")).thenReturn(response);
+        List<CredentialIssuerConfig> result = configurationService.getCredentialIssuers();
+
+        assertEquals(2, result.size());
```

Comment:
We might need to test that the two list items fields returned are the two we expect.